packages/webpack/LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2017 Honeybadger Industries LLC
```

Comment:
Can we delete this? Considering we have the LICENSE in the root folder of the monorepo?

Reply:
I'd guess so, but not sure.

Reply:
I'd guess so, but not sure.

Reply:
My guess is that it may be necessary when publishing the package on npm 🤔.

Reply:
Yeah, we want to make sure the license is included in each package (might want to also double-check the others).